ivy_tests/test_ivy/test_frontends/test_jax/test_nn/test_non_linear_activations.py

```diff
@@ -169,10 +169,12 @@ def test_jax_leaky_relu(
 @handle_frontend_test(
     fn_tree="jax.nn.gelu",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float"),
-        large_abs_safety_factor=8,
-        small_abs_safety_factor=8,
-        safety_factor_scale="log",
+        available_dtypes=helpers.get_dtypes("float_and_complex"),
+        large_abs_safety_factor=1,
+        small_abs_safety_factor=1,
+        safety_factor_scale="linear",
+        min_value=-1e4,
+        max_value=1e4,
```

Comment:
What's the reason behind these?

Reply:
the complex numbers  generated with large safety factor were too large and numerically unstable also sometimes It generates nan values  so I decided to lower the min and max value to make the tests consistent across different frameworks

Reply:
Right but
```
large_abs_safety_factor
    A safety factor of 1 means that all values are included without limitation,
```
the min and max make sense though

Reply:
I know that's odd but it worked for me

Reply:
Fair enough